integration-tests/jiva/node-stickiness/jiva_sc_resource.yaml

```diff
@@ -0,0 +1,12 @@
+# Creates jiva-single-replica storage class
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: jiva-single-replica
+  annotations:
+    openebs.io/cas-type: jiva
+    cas.openebs.io/config: |
+      - name: ReplicaCount
+        value: "1"
+provisioner: openebs.io/provisioner-iscsi
+---
```

Comment:
I would not prefer yaml but a package dedicated for storage class which we can use to build SC objects using builder functions. Imagine we are supposed to write BDDs with different variations of SC.

Reply:
cc @AmitKumarDas

Reply:
Done